processor/stash/stash.go

```diff
@@ -153,16 +156,15 @@ func (st *HandleT) storeErrorsToObjectStorage(jobs []*jobsdb.JobT) (errorJob []E
 	jobsPerWorkspace := lo.GroupBy(jobs, func(job *jobsdb.JobT) string {
 		return job.WorkspaceId
 	})
-	gzWriter := fileuploader.NewGzMultiFileWriter()
-	dumps := make(map[string]string)
+	writerMap := make(map[string]misc.GZipWriter)
 
 	errorJobs := make([]ErrorJob, 0)
 
 	ctx := context.Background()
 	for workspaceID, jobsForWorkspace := range jobsPerWorkspace {
 		preferences, err := st.fileuploader.GetStoragePreferences(ctx, workspaceID)
 		if err != nil {
-			st.logger.Errorf("Skipping Storing errors for workspace: %s since no storage preferences are found", workspaceID)
+			st.logger.Errorn("Skipping Storing errors for workspace since no storage preferences are found", logger.NewStringField("workspaceID", workspaceID))
```

Comment:
We can leverage obskit here. Similar for other logs as well.
```suggestion
			st.logger.Errorn("Skipping Storing errors for workspace since no storage preferences are found", obskit.WorkspaceID(workspaceID))
```